packages/cozy-client/src/helpers/settings.js

```diff
@@ -0,0 +1,182 @@
+import CozyClient from '../CozyClient'
+import fetchPolicies from '../policies'
+import { Q } from '../queries/dsl'
+
+const defaultFetchPolicy = fetchPolicies.olderThan(60 * 60 * 1000)
```

Comment:
I'm thinking about this default fetch policy. Maybe it is too long ? I do not have any use case now to explain (😅) but I have the feeling that with 1 hour we can get outdated setting value where we de not want outdated setting value.

Maybe it should be short but enough to avoid 2 requests when doing a getSetting + saveAfterFetchSetting? Like 10 seconds?